src/test/assembly/stack-probes.rs

```diff
@@ -0,0 +1,42 @@
+// min-llvm-version: 11.0.1
+// revisions: x86_64 i686
+// assembly-output: emit-asm
+//[x86_64] compile-flags: --target x86_64-unknown-linux-gnu
+//[i686] compile-flags: --target i686-unknown-linux-gnu
```

Comment:
This should definitely test at least all T1 targets, while also opportunistically adding checks for T2 and T3 targets.

Reply:
That said, I'm fine with a task being filled to flesh this test out after this lands as well.

Reply:
We'll need to find a way to specify targets that *only* support inline probes, since we only ever had `__rust_probestack` on i686/x86_64, and that may need min-LLVM too.

Reply:
I think it'd be fine to either duplicate the test for each version of LLVM or just have a test for the LLVM version that supports all the targets we're testing here.

But I conveniently forgot until your comment that current stack probing support only supports x86{,_64} anyway, and have missed that this PR does not change the support table in any meaningful way (i.e. it does not change any target to enable stack probes).

Reply:
I meant min-LLVM not just for testing, but to enable it at all in the target spec. PowerPC and SystemZ have support in LLVM 11, but I haven't tested that with Rust. AArch64 was going to aim for LLVM 12, and who knows for the rest.

Reply:
Ah I see!

Reply:
It might also be harmless to enable it on arches that don't actually support it, I'm not sure.